ouroboros-consensus/test-consensus/Test/Consensus/Ledger/Byron.hs

```diff
@@ -224,6 +226,7 @@ test_golden_ChainState = goldenTestCBOR
     secParam = SecurityParam 2
     windowSize = CS.WindowSize 3
     signers = map (`CS.PBftSigner` CC.exampleKeyHash) [1..5]
+    hashBytes = mkHeaderHashBytesForTestingOnly (Lazy8.pack "test_golden_ChainState6")
```

Comment:
Why `test_golden_ChainState6`?

Reply:
It's so the "hash" is indeed "specific" to the EBB. It's likely overkill, but I think it conveys an important subtlety. I added signatures and a comment that try to emphasize that.